keyboards/xd75/keymaps/hybrid/keymap_french.c

```diff
@@ -0,0 +1,157 @@
+enum french_key {
```

Comment:
This file doesnt seem to be referenced at all, and should be removed.

Reply:
was a deprecated thing. removed